tests/SignalSlot/RemoveTranslationSlotTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\SignalSlot\RemoveTranslationSlot;
+use eZ\Publish\Core\SignalSlot\Signal\ContentService\RemoveTranslationSignal;
+use eZ\Publish\SPI\Persistence\Content\Location;
+use eZ\Publish\SPI\Persistence\Content\Location\Handler as LocationHandler;
+
+/**
+ * Test RemoveTranslationSlot for HttpCache.
+ */
+class RemoveTranslationSlotTest extends AbstractContentSlotTest
+{
+    protected $locationId = 43;
+    protected $parentLocationId = 45;
+
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Content\Location\Handler|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $locationHandlerMock;
+
+    /**
+     * Check if required signal exists due to BC.
+     */
+    public static function setUpBeforeClass()
+    {
+        if (!class_exists(RemoveTranslationSignal::class)) {
+            self::markTestSkipped('RemoveTranslationSignal does not exist');
+        }
```

Comment:
> Buf if you'd feel safer with string FQCN, I can fix that :)

no, it's fine like this, it's how it is supposed to work actually. Use is not an import :)